src/pages/Dashboard/Versions/Detail/index.jsx

```diff
@@ -0,0 +1,471 @@
+    await modify({
+      version_id: version.version_id,
+      description
+    });
+
+    const result = appVersionStore.createResult;
+    if (!(result && result.err)) {
+      appVersionStore.info(t('Update log successful'));
+    }
+  };
+
+  resetLog = () => {
+    const { appVersionStore } = this.props;
+    appVersionStore.description = '';
+  };
+
+  toggleReason(record) {
+    record.isExpand = !record.isExpand;
+  }
+
+  handleVersion = async (handleType, noDailog) => {
+    const { appVersionStore, match } = this.props;
+    const { appId, versionId } = match.params;
+
+    if (!noDailog && (handleType === 'delete' || handleType === 'submit')) {
+      appVersionStore.showTypeDialog(handleType);
+    } else {
+      const result = await appVersionStore.handle(handleType, versionId);
+
+      // after operate will has new audit
+      if (!(result && result.err)) {
+        await appVersionStore.fetchAudits(appId, versionId);
+      }
+    }
+  };
+
+  deleteVersion = async () => {
+    const { appVersionStore, match, history } = this.props;
+    const { versionId, appId } = match.params;
+    const result = await appVersionStore.handle('delete', versionId);
+
+    if (!(result && result.err)) {
+      setTimeout(() => history.push(`/dashboard/app/${appId}/versions`), 1000);
+    }
+    appVersionStore.hideModal();
+  };
+
+  renderHandleMenu = () => {
+    const { appVersionStore, match, t } = this.props;
+    const { appId, versionId } = match.params;
+    const { version, showTypeDialog } = appVersionStore;
+    const deleteStatus = ['draft', 'rejected', 'passed'];
+    const hasDeleteBtn = deleteStatus.includes(version.status);
+
+    return (
+      <div className="operate-menu">
+        <Link to={`/dashboard/app/${appId}/deploy/${versionId}`}>
+          {t('Deploy App')}
+        </Link>
+        {hasDeleteBtn && (
+          <span onClick={() => showTypeDialog('delete')}>{t('Delete')}</span>
+        )}
+      </div>
+    );
+  };
+
+  renderDeleteDialog = () => {
+    const { appVersionStore, t } = this.props;
+    const { isDialogOpen, hideModal } = appVersionStore;
+
+    return (
+      <Dialog
+        title={t('Delete Version')}
+        isOpen={isDialogOpen}
+        onSubmit={this.deleteVersion}
+        onCancel={hideModal}
+      >
+        {t('Delete Version desc')}
+      </Dialog>
+    );
+  };
+
+  renderNoteDialog = () => {
+    const { appVersionStore, t } = this.props;
+    const { isDialogOpen, hideModal } = appVersionStore;
+
+    return (
+      <Dialog
+        title={t('Note')}
+        isOpen={isDialogOpen}
+        onSubmit={() => this.handleVersion('submit', true)}
+        onCancel={hideModal}
+        okText={t('Submit review')}
+        cancelText={t('Continue testing')}
+      >
+        <div className={styles.noteContent}>
+          <p>
+            在正式提交审核之前，请确认你的应用已经通过了以下基本功能的测试：
+          </p>
+          <div className={styles.noteList}>
```

Comment:
Why not use `ol` and `li`?

Reply:
There is mock data for show effect, will be revised later.